python/felis/db/utils.py

```diff
@@ -220,13 +220,13 @@ def __init__(self, metadata: MetaData, engine: Engine | MockConnection):
         self.metadata = metadata
         self.conn = ConnectionWrapper(engine)
 
-    def create_if_not_exists(self) -> None:
+    def initialize(self) -> None:
         """Create the schema in the database if it does not exist.
 
         Raises
         ------
         ValueError
-            If the database is not supported.
+            If the database is not supported or it already exists.
```

Comment:
The description above says "create the schema if it does not exist" but this text says it's a ValueError if it already exists. Which is it?

PS General comment that the style guide says a raises block test should start with `Raised if...`

Reply:
They shouldn't be mutually exclusive. Does the wording seem unclear?

The `--initialize` flag should be used if you expect the database to not exist. If it already exists, then the flag is not needed. An error will also be raised, because you are expecting to create and update an empty schema/database and don't want to use an existing one.

Reply:
> style guide says a raises block test should start with `Raised if...`

I will update this though I have a lot of other `Raises` docstrings that don't follow this style.

Reply:
Right. Sorry. It's fine. I was reading too quickly.

Reply:
@timj I updated all the `Raises` sections in the docstrings to follow the recommended style.